WordPress/Classes/ViewRelated/Reader/WPTableViewHandler.m

```diff
@@ -522,6 +522,7 @@ - (void)controller:(NSFetchedResultsController *)controller
         {
             [self invalidateCachedRowHeightAtIndexPath:indexPath];
             [self.tableView reloadRowsAtIndexPaths:@[indexPath] withRowAnimation:self.updateRowAnimation];
+            self.indexPathSelectedAfterUpdates = self.tableView.indexPathForSelectedRow;
```

Comment:
Could you help me understand the reason for this line? It looks like it would conflict with the logic under the `NSFetchedResultsChangeMove` block so I'm not sure its a good change.